scss/bootstrap-debugger.scss

```diff
@@ -0,0 +1,3 @@
+$enable-debugger-classes: true; // stylelint-disable-line scss/dollar-variable-default
+
+@import "bootstrap";
```

Comment:
Personally, I would include only `scss/_debug.scss` (and the right imports for the compilation), so that `bootstrap-debugger.css` contains just the debugging elements. And then, no need for an RTL version.

And the other `scss/bootstrap-*.scss` wouldn't include `@import "debug"`.

That way, everybody is using the same `bootstrap*.css` files (with the same sha384) and could enable the debugging part via Sass compilation and `$enable-debugger-classes`, or via CDN by importing `bootstrap-debugger.css` after (I suppose) the regular Bootstrap CSS files (RTL or not).

Reply:
I understand your point, I'va made the changes but for me, we should either have a `$enable-debugger-classes` and use `@import` in the good files or have no option and so let the users decide when they import it or not.